daq/test_modules/ipaddr_module.py

```diff
@@ -63,7 +63,7 @@ def _get_lease_time(self):
     def _set_timeout(self):
         if not self._lease_time_seconds:
             return
-        self._timeout = datetime.now() + timedelta(seconds=self._lease_time_seconds)
+        self._timeout = datetime.now() + timedelta(seconds=self._lease_time_seconds + 30)
```

Comment:
@grafnu what do you think about this?

Reply:
I'd like to know "why" -- DHCP should normally be re-requested after 1/2 the lease time, so why does the timeout need to be more?  And then -- 30 seems somewhat arbitrary and a bit longer than I'd expect for a fudge factor.  Does this work sufficiently with sometime like 5s, which is more what I'd expect...   OTOH, I'm not sure what the negative impact is here of this -- e.g. does this timeout get regularly used, or only in actual failure cases?